### ISSUE TITLE
[BUG] The latest release has a bug with the package name resulting in installation errors

### ISSUE THREAD

**Issue by jackton1** [rollbar/pyrollbar#383]
### Steps to replicate
- `pip install rollbar==0.16.0`

@bxsx 

```bash
File "/private/var/folders/bs/22hly3997_3f0zfqvh094jp00000gn/T/pip-build-env-s8nco__a/overlay/lib/python3.6/site-packages/poetry/core/masonry/api.py", line 44, in prepare_metadata_for_build_wheel
        builder = WheelBuilder(poetry)
      File "/private/var/folders/bs/22hly3997_3f0zfqvh094jp00000gn/T/pip-build-env-s8nco__a/overlay/lib/python3.6/site-packages/poetry/core/masonry/builders/wheel.py", line 57, in __init__
        super(WheelBuilder, self).__init__(poetry, executable=executable)
      File "/private/var/folders/bs/22hly3997_3f0zfqvh094jp00000gn/T/pip-build-env-s8nco__a/overlay/lib/python3.6/site-packages/poetry/core/masonry/builders/builder.py", line 89, in __init__
        includes=includes,
      File "/private/var/folders/bs/22hly3997_3f0zfqvh094jp00000gn/T/pip-build-env-s8nco__a/overlay/lib/python3.6/site-packages/poetry/core/masonry/utils/module.py", line 64, in __init__
        "No file/folder found for package {}".format(name)
    poetry.core.masonry.utils.module.ModuleOrPackageNotFound: No file/folder found for package pyrollbarfastapi
    ----------------------------------------

```

`pyproject.toml`

```py
[tool.poetry]
name = "pyrollbarfastapi"
version = "0.1.0"
description = ""
authors = ["Bart Skowron <bart.skowron@rollbar.com>"]

[tool.poetry.dependencies]
python = "~3.8"
uvicorn = {extras = ["standard"], version = "^0.13.4"}
requests = "^2.25.1"
six = "^1.15.0"
python-multipart = "^0.0.5"
fastapi = "0.65.1"
httpx = "^0.18.1"

[tool.poetry.dev-dependencies]
unittest2 = "^1.1.0"
WebOb = "^1.8.7"
pyramid = "^2.0"
black = "^20.8b1"
isort = "^5.7.0"
asynctest = "^0.13.0"
Twisted = "^21.2.0"
treq = "^21.1.0"
Flask = "^1.1.2"
blinker = "^1.4"
mypy = "^0.812"
Quart = "^0.15.0"
pytest = "^6.2.4"
bottle = "^0.12.19"
sanic = "^21.3.4"
falcon = "^3.0.1"
sentry-sdk = {extras = ["flask"], version = "^1.1.0"}
channels = "^3.0.3"

[build-system]
requires = ["poetry-core>=1.0.0"]
build-backend = "poetry.core.masonry.api"

[tool.isort]
profile = "black"

[tool.black]
line-length = 88
include = '\.pyi?$'
exclude = '''
/(
    \.git
  | \.hg
  | \.mypy_cache
  | \.tox
  | \.venv
  | _build
  | buck-out
  | build
  | dist
)/
'''

```

**Comment by bxsx** [comment]
Hey, thanks for the report and sorry for the trouble. I have just released v0.16.1 with the fix.

**Comment by jackton1** [comment]
Thanks @bxsx
